src/query/management/src/udf/errors.rs

```diff
@@ -12,30 +12,85 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::fmt::Display;
+
 use databend_common_exception::ErrorCode;
 use databend_common_meta_types::MetaError;
 
 use crate::errors::TenantError;
 
+/// UDF logic error, unrelated to the backend service providing UDF management, or dependent component.
 #[derive(Clone, Debug, thiserror::Error)]
 pub enum UdfError {
-    #[error("TenantError: '{0}'")]
-    TenantError(#[from] TenantError),
+    #[error("UDF not found: '{tenant}/{name}'; {context}")]
```

Comment:
Add tenant to error message may be not a good idea? Say in the cloud, it will raise to the user.

Reply:
Got it. Let me review the changes again.